app/controls/open-browser.js

```diff
@@ -1,23 +1,24 @@
 import ipc from 'electron-safe-ipc/host'
 import {apiAddrToUrl} from './utils'
-import {getIPFS} from './../init'
+import {logger, getIPFS} from './../init'
```

Comment:
No because it is not coming from the config but from the main index file that starts everything up